src/parcsr_ls/par_ilu.h

```diff
@@ -39,7 +39,8 @@ typedef struct hypre_ParILUData_struct
    hypre_ParCSRMatrix      *P;
    hypre_Vector            *Ftemp_upper;
    hypre_Vector            *Utemp_lower;
-   HYPRE_Int               *A_diag_fake;//fake diagonal, used to pretend that the diagonal matrix is empty
+   HYPRE_Int
+   *A_diag_fake;//fake diagonal, used to pretend that the diagonal matrix is empty
```

Comment:
It appears that having // immediately after the ; is causing this to be broken up strangely. It probably should be manually fixed.

Reply:
I fixed this one.  In general, when the line is too long, astyle doesn't seem to make the best decision about how to break it.  I think we'll just have to manually fix these as we come across them.  Thanks!